### PR TITLE
Fix knockout templates for jQuery 3

### PR DESCRIPTION
**Pull request by orangejenny**
Same issue as https://github.com/dimagi/commcare-hq/pull/13677

Grepped for places we use jQuery to create a div that uses a knockout template, looks like it's just these two files.

@dannyroberts

### CODE REVIEW

**Comment by orangejenny** [comment]
@dannyroberts could you review? jQuery 3 is ready to go.

File: corehq/apps/app_manager/static/app_manager/js/graph-config.js [hunk 1]
```diff
@@ -43,7 +43,8 @@ hqDefine('app_manager/js/graph-config.js', function () {
             };
 
             // Load the modal with the copy
-            var $modalDiv = $('<div data-bind="template: \'graph_configuration_modal\'"></div>');
+            var $modalDiv = $(document.createElement("div"));
+            $modalDiv.attr("data-bind", "template: \'graph_configuration_modal\'");
```
Comment (dannyroberts) [inline corehq/apps/app_manager/static/app_manager/js/graph-config.js]: I think the escaping in `\'` is no longer necessary

File: corehq/apps/hqwebapp/static/hqwebapp/js/key-value-mapping.js [hunk 2]
```diff
@@ -189,7 +189,8 @@ uiElement.key_value_mapping = function (o) {
     m.openModal = function () {
         // create a throw-away modal every time
         // lets us create a sandbox for editing that you can cancel
-        var $modalDiv = $('<div data-bind="template: \'key_value_mapping_modal\'"></div>');
+        var $modalDiv = $(document.createElement("div"));
+        $modalDiv.attr("data-bind", "template: \'key_value_mapping_modal\'");
```
Comment (dannyroberts) [inline corehq/apps/hqwebapp/static/hqwebapp/js/key-value-mapping.js]: ditto

**Review by dannyroberts — APPROVED** [review]

**Comment by dannyroberts** [comment]
Looks good to me (though might address the one comment if you agree)

File: corehq/apps/app_manager/static/app_manager/js/graph-config.js (hunk 1 above)
Comment (orangejenny) [inline corehq/apps/app_manager/static/app_manager/js/graph-config.js]: indeed, updated

**Pull request merged**